README.md

```diff
@@ -29,3 +29,19 @@ func main() {
 For full API documentation, including bypass mechanisms, see [pkg.go.dev](https://pkg.go.dev/filippo.io/csrf).
 
 For more information on this approach, see [the standard library proposal](https://go.dev/issue/73626).
+
+## github.com/gorilla/csrf compatibility
+
+The `filippo.io/csrf/gorilla` package provides a drop-in replacement for the
+`github.com/gorilla/csrf` package. It implements the same API, but uses the
+modern Fetch metadata headers instead of tokens and cookies.
```

Comment:
as a person who might in fact need CSRF protections, but does not have a detailed understanding, I might benefit from a paragraph or so telling me what the implications of this might be. how would i know whether i was using tokens and cookies? is there a likely or common use case where this replacement wouldn't work? if it "didn't work", would i be creating security holes, or would i have a secure site where some functionality is no longer available? i think that last question is probably the most important, and the implication is probably obvious to security people, but i am not a security people.

Reply:
The package docs try harder at giving more context, and there's specifically a paragraph that I think answers your questions, but ultimately if you adopted github.com/gorilla/csrf blindly to solve CSRF, this package will be fine, and if you are doing weirder/less standard things you either have the background to know what they are, or we can't help.